.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,17 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Triage Reviewer Checklist
```

Comment:
```suggestion
## Reviewer Checklist
```

? does it matter if it is a "Triage" or not, seems like these are just good items to check for any reviewer at all times?